ui/src/components/easter/hero/HeroContainer.js

```diff
@@ -106,29 +119,42 @@ const HeroContainer = () => {
                     target="_blank"
                   >
                   
-                  <LinkOverlay href="https://www.youtube.com" target="_blank">
+                  <LinkOverlay target="_blank" onClick = {()=> setOpen(true)}>
```

Comment:
can simplify to `onClick={setOpen(true)}`

Reply:
cannot be simplified as it will read error
![41e461d2-518a-483e-8253-13936c9e6cfe](https://user-images.githubusercontent.com/33675629/159122808-073c6471-1976-45a1-a7ed-c9541bffe23b.jpg)